.github/workflows/subtree.yml

```
@@ -29,7 +30,28 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: nxtlvlsoftware/git-subtree-action@1.1
+      - uses: pnpm/action-setup@v2.2.4
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
```

```suggestion
          node-version: 18.x
          cache: 'pnpm'
```

Idk if cache Will help when it should generate a lockfile?

---

idk